plexapi/video.py

```diff
@@ -711,6 +740,13 @@ def seasonEpisode(self):
         """ Returns the s00e00 string containing the season and episode. """
         return 's%se%s' % (str(self.seasonNumber).zfill(2), str(self.index).zfill(2))
 
+    @property
+    def hasIntroMarker(self):
+        """ Returns True if this episode has an intro marker in the xml. """
+        if not self.isFullObject():
+            self.reload()
+        return bool(self.markers)
```

Comment:
I would only check for `intro` markers in case Plex decides to add different marker types in the future.

```python
any(marker.type == 'intro' for marker in self.markers)
```